README.md

```diff
@@ -23,6 +23,17 @@ To run the site locally:
    ```
 4. Open http://localhost:8080
 
+### Accessiblity Scanning
+We use `pa11y-ci` is used to scan for accessibility issues. The scan runs as part of
+our CI setup (see the [pull-request.yml workflow](.github/workflows/pull-request.yml))
+on every pull request, but it can also be run locally. To run locally, type:
+
+```
+npm run test:pa11y-ci
+```
+
+Note that running `pa11y-ci` inside the docker container may not always work.
+
```

Comment:
What happen when docker doesnt work? Is there any workaround?

Reply:
As far as I can tell it won't work inside the container, only running locally or in the GH action.